app/src/test/java/net/gsantner/markor/format/zimwiki/ZimWikiReplacePatternGeneratorTests.java

```diff
@@ -147,7 +146,7 @@ private void assertCorrectReplacement(String original, String expectedReplacemen
 
         private String replaceWithFirstMatchingPattern(List<TextActions.ReplacePattern> replacePatterns, String input) {
             for (TextActions.ReplacePattern replacePattern : replacePatterns) {
-                Matcher matcher = replacePattern.searchPattern.matcher(input);
```

Comment:
Creating a matcher is surprisingly expensive compared to resetting it with a new input